en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
 
 You can add logic and actions in different patterns, depending on what your flow needs to do.
 
+== Designate Error Handling 
+
+When executing a flow, you can watch for errors in connection or logic in specific steps of your flow and then designate logic to handle the errors that may occur. You can also designate steps to complete additional actions or to send notifications in the event of an error. 
+
+[[access-error-handling]]
+=== Access Error Handling 
+
+To access the Error Handler:
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow.
+. If you create a new flow:
+.. Select the connector or system to start your flow.
+.. Select and configure a trigger. 
+.. Select and configure an action.
+.. Directly before the step to which you want to add an Error Handler, click the plus button. 
+.. In the *What Should Start This Flow?* window, click *Error Handler (Beta)*.
+. If you add a step to an existing flow:
+.. Open your flow.
+.. Directly before the step to which you want to add an Error Handler, click the plus button. 
+.. In the *Next Step* window, click *Error Handler (Beta)*. +
+
+[[watch-a-step]]
+=== Watch a Step 
+
+You can watch specific steps for errors in connection or logic. If an error occurs, the flow stops normal processing and begins processing the steps in the Error Handler. You can add one or more steps to watch in this section.
+
+To watch a step: 
+
+. <<access-error-handling,Access the Error Handler in your flow>>.
+.  In the *Error Handler* card, in the *Watch* section, click the plus sign and select the action in your flow that you want to watch or drag the step that you want to watch into the *Watch* section. +
+The selected action appears in the *Watch* section. If an error occurs, the flow moves to the actions selected in the *On Error* section.
+. To add additional actions to watch, repeat the previous step.
+
+=== Configure Composer to Handle Errors 
```

Comment:
Should this be an H2?

Reply:
I don't think so. All of these subsections are part of the larger H2 "Configure Error Handling". 

There are plans to break up these monster topics a bit in the future.